### ISSUE TITLE
Can't zoom out after programmatic zoom when using wheel zoom

### ISSUE THREAD

**Issue by ebencollins** [naver/billboard.js#2157]
## Description
With drag zoom, the "reset zoom" button appears, letting the user reset to viewing the full chart. With wheel zoom, nothing happens when trying to scroll out; it's still possible to pan and zoom in, but it's not possible to zoom out past where the programmatic zoom was set.

My goal is to programmatically zoom in to a region, then allow the user to zoom back out to see the full chart.

Setting the zoom extent to `[0, 10]` instead of `[1, 10]` allows zooming out to the full chart, but also allows [zooming out past 0](https://i.imgur.com/EaAc4MN.png) - even with `x.min` set to `0`

## Steps to check or reproduce
Run the following chart, then try to zoom out.
https://stackblitz.com/edit/js-xn4cmq

```
var chart = bb.generate({
  data: {
    columns: [
	["sample", 30, 200, 100, 400, 150, 250, 150, 200, 170, 240, 350, 150, 100, 400, 150, 250, 150, 200, 170, 240, 100, 150, 250, 150, 200, 170, 240, 30, 200, 100, 400, 150, 250, 150, 200, 170, 240, 350, 150, 100, 400, 350, 220, 250, 300, 270, 140, 150, 90, 150, 50, 120, 70, 40]
    ],
    type: "line",
  },
  zoom: {
    enabled: true, 
    // extent: [0, 10], // this allows zoom out past programmatic zoom window, but allows zoom well past beginning of chart
    x: {
      // this seems to have no effect
      min: 0
    },
  },
  bindto: "#zoom"
});

chart.zoom([20, 30]);
```

**Comment by netil** [comment]
you can reset zoom by `.unzoom()`. drag zoom type provide `resetZoom` button functionality, because unlike the wheel type, there's no way to zoom by user interaction.
Maybe providing same reset button interface as drag can enhance in this scenario.

**Comment by ebencollins** [comment]
> you can reset zoom by `.unzoom()`. drag zoom type provide `resetZoom` button functionality, because unlike the wheel type, there's no way to zoom by user interaction.

My goal isn't to reset the zoom to original (as `unzoom()` would do), but to allow the user to zoom out from the window set by `.zoom`. For example, after `.zoom([30, 40])`, the user may want to view `[25, 45]` by scrolling out out slightly. Currently, nothing happens when attempting to zoom out.

The issue I have is that without setting the extent, the user can never zoom out past the level set by `.zoom`, and if I set extent to something like `[0, 10]`, it lets the user zoom out far past the beginning of the chart even when `x.min` is `0` (though I might be misunderstanding the functionality of this option)

**Comment by ebencollins** [comment]
Seems to have been fixed in 3.1.3
